brands/amenity/social_centre.json

```diff
@@ -63,6 +63,15 @@
       "official_name": "Independent Order of Odd Fellows"
     }
   },
+  "amenity/social_centre|Royal Canadian Legion" : {
+    "countrycodes": ["ca"],
```

Comment:
```suggestion
    "countryCodes": ["ca"],
```

Reply:
BTW: by running `npm run build` after you made the changes to the file, you get a direct "feedback" whether the data is correct or not. And it does also format the edited file to be consistent with the other data (e.g. it sorts the entries alphabetically)

Reply:
@bkissin If you're editing in a browser via Github.com (I am guessing you are), don't worry about the `npm run build` suggestion but do look to see if your Pull Request passes or fails the automated “checks”.  If you see at the bottom of the PR that “Travis CI build failed” this likely indicates that you made a syntax error.